include/swift/Driver/Job.h

```diff
@@ -230,9 +230,17 @@ class CommandOutput {
 class Job {
 public:
   enum class Condition {
+    // There was no input map, or the map makred this Job as dirty
```

Comment:
I don't think it's obvious what "input map" means here. Maybe something like "we have no information about any previous build"?

Reply:
Good catch! Fixed it to say both.